packages/server/src/api/controllers/static/index.ts

```diff
@@ -243,8 +327,45 @@ export const serveApp = async function (ctx: UserCtx<void, ServeAppResponse>) {
 
       const { head, html, css } = AppComponent.render({ props })
       const appHbs = loadHandlebarsFile(appHbsPath)
+
+      let extraHead = ""
+      const pwaEnabled = true // await pro.features.isPWAEnabled()
```

Comment:
This needs to be properly set

Reply:
I can't add the license on the feature branch right now as account portal has stopped deploys, so this is just temporary.